src/find_controls.cpp

```diff
@@ -181,10 +180,9 @@ std::ostream& operator<<(std::ostream& os, const FindMode::Values& v)
 
 FindMode::Values GetFindModeValueImpl2()
 {
-    const char* const p_asciz = miopen::GetStringEnv(MIOPEN_FIND_MODE{});
-    if(p_asciz == nullptr)
+    auto str = miopen::GetStringEnv(MIOPEN_FIND_MODE{});
```

Comment:
This seems like an `integer` type variable.

Reply:
This env could be parsed as a string or an integer. But it does map to an enum in the end.

Reply:
two reasons to support converting it to integer:
1. Integers are a natural fit for enums
2. We can avoid doing `strtoul` in multiple places and consolidate it in `env.hpp` where do error checking as well.

Reply:
If you'll observe lines 188-195
Reference by **name** of the enumerated value is permitted as well as the numerical representation.